### ISSUE TITLE
RuntimeError: CUDA failed with error CUDA driver version is insufficient for CUDA runtime version

### ISSUE THREAD

**Issue by SwAt1563** [SYSTRAN/faster-whisper#1115]
![image](https://github.com/user-attachments/assets/58883e54-dfa2-4d20-bf4a-ceee4c70e1d4)

![image](https://github.com/user-attachments/assets/df9a0c59-ce9b-467a-b4e7-587d4e09e159)

![image](https://github.com/user-attachments/assets/b614720f-6aea-4ece-ad17-4b363d48a7ca)

------

**Issue Summary:**
I'm attempting to run `faster-whisper` with GPU support on my device, which has CUDA 12.6 installed. The following package versions are being used:

- `faster-whisper==1.0.3`
- `ctranslate2==4.4.0`
- `wyoming==1.5.3`

The model runs successfully on the CPU; however, it does not utilize the GPU as expected. I am looking for a solution to enable GPU acceleration for `faster-whisper` using CUDA 12.6.

**Comment by jhj0517** [comment]
If you're using docker you can try changing 
https://github.com/SYSTRAN/faster-whisper/blob/814472fdbf7faf5d77d65cdb81b1528c0dead02a/docker/Dockerfile#L5

into
```
 RUN pip3 install faster-whisper torch --index-url https://download.pytorch.org/whl/cu124
```

Currently there's a strict version matrix between `torch` and `faster-whisper` : https://github.com/SYSTRAN/faster-whisper/issues/1086#issue-2612158224

**Comment by SwAt1563** [comment]
Thank you for your guidance, but it still doesn't work.

**Comment by jedmeister324** [comment]
Windows 10 pro, rtx3080, whisper-faster, the CUDNN files are installed and correct paths etc, torch installed, ctranslate on 4.4...... faster_whisper loads the model into gpu memory, but processing is done on cpu.  I've tried every single recommendation listed here and on other sites, uninstalled/reinstalled several times, but, faster_whisper still using CPU. 

I have another desktop on win11 with a RTX1070 -- I got that to work fine for GPU processing. 

There is an app called whisper-desktop , which is an EXE / gui frontend to whisper .... whisper-desktop runs on my win10/rtx3080 using the GPU processing.  I'm guessing this whisper-desktop has the correct settings built-in to the exe. 

So, if whisper-desktop works, what am I doing wrong to get whisper-faster using the GPU?
